docs/guide/clustering.md

```diff
@@ -130,13 +134,13 @@ doctrine:
     dbal:
         connections:
             dfs:
+                # configure these settings to match your database server
                 driver: pdo_mysql
-                host: 127.0.0.1
-                port: 3306
-                dbname: ezdfs
-                user: root
-                password: rootpassword
-                charset: UTF8MB4
+                charset: utf8mb4
+                default_table_options:
+                    charset: utf8mb4
+                    collate: utf8mb4_unicode_520_ci
+                url: mysql://root:rootpassword@127.0.0.1:3306/ezdfs
 
 # define the Flysystem handler
 oneup_flysystem:
```

Comment:
This configuration also was changed, 3 lines below - I don't have a way to mark it. See the original PR ezsystems/ezplatform#596. You also need to include `parameters` section at the top, but just `ibexa.platform.io.nfs.adapter.config`, since the other two params are not used for this snippet.